pkg/fqdn/matchpattern/matchpattern.go

```diff
@@ -70,6 +70,11 @@ func ToAnchoredRegexp(pattern string) string {
 	pattern = strings.TrimSpace(pattern)
```

Comment:
function doc comment needs to learn about `"**"` too

Reply:
Added.